can_tools/scrapers/official/AK/ak_vaccine.py

```diff
@@ -0,0 +1,262 @@
+        keep = df[["dt", "dose_num", "fips", "primarykeyid"]]
+        count_per_day = (
+            keep.groupby(["dt", "fips", "dose_num"])
+            .count()
+            .rename(columns={"primarykeyid": "count"})
+        )
+        initiated = self._get_total_initiated_by_day(count_per_day)
+        completed = self._get_total_completed_by_day(count_per_day)
+        # Don't know what dose_num = 3 means...?
+        other = (
+            count_per_day[count_per_day.index.get_level_values("dose_num") == 3]
+            .rename(columns={"count": "vaccine_other"})
+            .reset_index()
+            .set_index(["dt", "fips"])
+        )
+
+        out = (
+            initiated.join(completed["vaccine_completed"])
+            .join(other["vaccine_other"])
+            .drop(columns="dose_num")
+            .replace(np.nan, 0)
+            .reset_index()
+        )
+        out = self._fix_fips(out)
+        return out.dropna()
+    
+    def _get_total_initiated_by_day(self, df):
+        # Get a cumsum of all rows where dose_num == 1
+        initiated = (
+            df[df.index.get_level_values("dose_num") == 1]
+            .rename(columns={"count": "vaccine_initiated"})
+            .reset_index()
+            .set_index(["dt", "fips"])
+        )
+        return initiated
+
+    def _get_total_completed_by_day(self, df):
+        # Get a cumsum of all rows where dose_num == 2
+        completed = (
+            df[df.index.get_level_values("dose_num") == 2]
+            .rename(columns={"count": "vaccine_completed"})
+            .reset_index()
+            .set_index(["dt", "fips"])
+        )
+        return completed
+
+    def _rename_and_prep(self, data):
+        df = self.arcgis_jsons_to_df(data)
+        df = df.rename(
+            columns={
+                "RECIP_ID": "recipient_id",
+                "RECIP_SEX": "sex",
+                "RECIP_ETHNICITY": "ethnicity",
+                "DATA_DT": "dt",
+                "VAX_SERIES_COMPLETE": "fully_vaccinated",
+                "REGION_NUM": "region",
+                "BOROUGH_FIPS": "fips"
+            }
+        )
+        df.columns = df.columns.str.lower()
+        df.dt = pd.to_datetime(df.dt, unit="ms", origin="unix")
+
+        return df
+
+    def normalize(self, data):
+        df = self._rename_and_prep(data)
+        totals = self._get_total_doses_administered_by_day(df)
+        
+        # drop "other" for now
+        totals = totals.drop(columns={"vaccine_other"})
+        # melt daily/new data to long form
+        out = (
+            totals.melt(id_vars=["fips", "dt"], value_vars=self.variables.keys())
+            .dropna()
+        )
+        
+        out = (out.pipe(self.extract_CMU, cmu=self.variables)
+            .assign(
+                vintage=self._retrieve_vintage(),
+                dt=out['dt'].dt.date
+            )
+            .drop(["variable"], axis=1)
+        )
+        return out.rename(columns={'fips':'location'})
```

Comment:
```suggestion
        out = out.rename(columns={'fips':'location'})
        out = out.query("location != 2000").replace(2270, 2158)
        return out
```
The 2270 fips was renamed and now has a fips of 2158, so this updates that change. 

removed the `000` county so that the tests would pass, but not sure what the best course of action is.